src/coffee/extend/ForceDirectedGraph.coffee

```diff
@@ -0,0 +1,154 @@
+Function::property = (prop, desc) ->
+  Object.defineProperty @prototype, prop, desc
+
+
+# Hacky Edge implementation
+class Edge
+  constructor: (source, target) ->
```

Comment:
The `Edge` implementation holds a reference to the `source` and `target`. The `Line`/`Point` implementation currently just copies the coordinates from `source`/`target`.

Two solutions I can currently think of:
* We change `Graph.addEdge(Edge edge)` to `Graph.addEdge(Point, Point)` or `Graph.addEdge([Point, Point])`
* Let `Pair` also accept references to `Point`.